content/en/agent/autodiscovery/management.md

```diff
@@ -0,0 +1,133 @@
+---
+title: Container Discovery Management
+kind: documentation
+further_reading:
+- link: "/agent/autodiscovery/integrations"
+  tag: "Documentation"
+  text: "Create and load an Autodiscovery Integration Template"
+- link: "/agent/autodiscovery/ad_identifiers"
+  tag: "Documentation"
+  text: "Match a container with the corresponding Integration Template"
+---
+
+Datadog Agent auto-discovers all containers available by default. To restrict its discovery perimeter and limit data collection to a subset of containers only, include or exclude them through a dedicated configuration.
+
+**Note**: The `docker.containers.running`, `.stopped`, `.running.total` and `.stopped.total` metrics are not affected by these settings and always count all containers. This does not affect your per-container billing.
+
+If running the Agent as a binary on a host, configure your Autodiscovery perimeter with the [Host Agent](?tab=host-agent) tab instructions. If running the Agent as a Container configure your Autodiscovery perimeter with the [Containerized Agent Tab](?tab=containerized-agent) instructions.
+
+## Exclude containers
+
+Exclude containers from the Agent Autodiscovery perimeter with an exclude rule based on their `name` or `image` to collect **NO DATA** out of it. If a container matches an exclude rule, it won't be included unless it first matches an include rule.
+
+**Note**: Exclude rules support Regexp, and are defined as a list of comma-separated strings.
+
+{{< tabs >}}
+{{% tab "Host Agent" %}}
+
+To remove a given docker container with the image `<IMAGE_NAME>` from autodiscovery add the following configuration block in the [Agent `datadog.yaml` configuration file][1]:
```

Comment:
should Autodiscovery be capitalized? I'm not sure, but either way, you should make it more consistent.